spec/lib/tracked_revisions_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_dependency "#{Rails.root}/lib/analytics/course_statistics.rb"
+
+describe 'TrackedRevisions' do
+  let(:course) { create(:course, start: '30-05-2019'.to_date, end: '31-07-2019'.to_date) }
+  let(:user) { create(:user, username: 'Textorus') }
```

Comment:
I recommend choosing a user who is less active than this one, and using a date farther in the past.

One consideration is that the course update process pulls all the revisions from the course start date through the present, so over time the query could get slower if it's a very active user. The total edits over all time are also a factor in how quickly the queries go.